packages/venia-concept/src/actions/cart/__tests__/asyncActions.spec.js

```diff
@@ -897,6 +897,32 @@ describe('toggleCart', () => {
         expect(dispatch).not.toHaveBeenCalled();
     });
 
+    test('toggleCart thunk opens the drawer and cart is empty', async () => {
+        getState.mockReturnValueOnce({
+            app: { drawer: 'cart' },
+            cart: { details: { items_count: 0 } }
+        });
+        await toggleCart()(...thunkArgs);
+
+        expect(dispatch).toHaveBeenNthCalledWith(1, expect.any(Function));
+        expect(dispatch).toHaveBeenCalledTimes(1);
+    });
+
+    test('toggleCart thunk opens the drawer and refreshes the cart', async () => {
+        getState.mockReturnValueOnce({ app: { drawer: 'cart' }, cart: {} });
+        await toggleCart()(...thunkArgs);
+        expect(dispatch).toHaveBeenNthCalledWith(1, expect.any(Function));
+        expect(dispatch).toHaveBeenCalledTimes(1);
```

Comment:
Let's split these assertions out into their own test. Something like "its thunk opens the drawer when the cart has no details".

Reply:
I've tried to split it but I have a little bit confused with test cases. So, Can you please help with this?